WebRoot/skins/_base/base4/skin.css

```diff
@@ -997,3 +997,33 @@ div.ZmHtmlEditor .TxtAreaDrag {
 	padding-right: 0;
 }
 
+/* Align columns for group edit */
+.ZmGroupMembersWrapper.DwtChooserListViewCustomStyle .DwtChooserListView .DwtListView-Column {
+    padding-bottom: 12px;
+}
+
+/* Custom DwtChooserListView Style */
+.DwtChooserListViewCustomStyle .DwtChooserListView {
+    @ChooserListViewCustomStyle@
+}
+
+.DwtChooserListViewCustomStyle .DwtChooserListView .DwtListView-ColHeader {
+    @ChooserListViewCustomStyleHeader@
+}
+
+.DwtChooserListViewCustomStyle .DwtChooserListView .DwtListView-ColHeader .DwtListHeaderItem-label {
+    @ChooserListViewCustomStyleHeaderLabel@
+}
+
+.DwtChooserListViewCustomStyle .DwtChooserListView .DwtListView-Column {
+    @ChooserListViewCustomStyleColumn@
+}
+
+.DwtChooserListViewCustomStyle .DwtChooserListView .DwtListView-Rows {
+    @ChooserListViewCustomStyleContainer@
+}
+
+/* For DL edit, hide location */
+.ZmDlViewWrapper .ZmContactHeaderLocation {
```

Comment:
This CSS declaration has been removed and functionality is handled at template level as per review comment from @tarangkhandelwal 

Thanks